kyuubi-main/src/test/scala/org/apache/kyuubi/operation/WithKyuubiServer.scala

```diff
@@ -19,22 +19,33 @@ package org.apache.kyuubi.operation
 
 import java.nio.file.Files
 
-import org.apache.kyuubi.{KyuubiFunSuite, Utils}
+import org.apache.kyuubi.Utils
 import org.apache.kyuubi.config.KyuubiConf
 import org.apache.kyuubi.config.KyuubiConf._
+import org.apache.kyuubi.config.internal.Tests.TESTING_YARN_CONF_DIR
 import org.apache.kyuubi.ha.HighAvailabilityConf.{HA_ZK_ACL_ENABLED, HA_ZK_QUORUM}
 import org.apache.kyuubi.server.KyuubiServer
 import org.apache.kyuubi.zookeeper.{EmbeddedZookeeper, ZookeeperConf}
 
-trait WithKyuubiServer extends KyuubiFunSuite {
+trait WithKyuubiServer extends WithMiniYarnCluster {
```

Comment:
We don't need to make all test extend `WithMiniYarnCluster `. We can choose to use it if some suites need.

Reply:
will try it later